python/invoke_release/tasks.py

```diff
@@ -1359,7 +1363,12 @@ def release(_, verbose=False, no_stash=False):
 
         if USE_PULL_REQUEST:
             _checkout_branch(verbose, current_branch_name)
-
+            try:
+                os.environ["GITHUB_TOKEN"]
+            except KeyError:
+                _standard_output("GITHUB_TOKEN env var not set. Unable to open a github PR")
+            else:
+                open_pull_request(branch_name, BRANCH_MASTER, release_version, os.environ["GITHUB_TOKEN"])
```

Comment:
We can store `os.environ["GITHUB_TOKEN"]` in a var in the `try` statement and in here just use the var, instead of accessing the element again.